specs/digit_match_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require_relative '../lib/digit_match'
+
+describe "digit match" do
+  describe "basic tests" do
+    describe "test 1" do
```

Comment:
Same as the other repos, we should probably remove the `describe` blocks that are immediate parents of individual tests.